cmd/create.go

```diff
@@ -110,19 +109,20 @@ func newCreateConfig(args []string) createConfig {
 // Skipped if not in an interactive terminal (non-TTY), or if --confirm (agree to
 // all prompts) was not explicitly set.
 func (c createConfig) Prompt() createConfig {
-	name := deriveName(c.Name, c.initConfig.Path)
```

Comment:
I am pretty sure that moving this name derivation code below the prompt, means the confirmation prompt will not display the value that will actually be used.  

By deriving `name` first, as the calculated derived and storing it in a temporary variable, it allows the prompt to display the dervied value that would be used if the client library were to be invoked directly (it would derive the value).

Reply:
Perhaps this is not the case, as it appears this logic is actually now just moved into the initConfig directly.  I presume this was to make both `create` and `init` work the same?  My comment above may therefore not be applicable.